network/dag/sigverifier.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	"github.com/lestrrat-go/jwx/jwa"
 	"github.com/lestrrat-go/jwx/jws"
-	"github.com/nuts-foundation/nuts-node/crypto"
+	"github.com/nuts-foundation/nuts-node/vdr/types"
```

Comment:
I still find the "types" package an odd thing, especially because not all engines have it (some manage to be a dependency to other engines without it).

"packages should describe the functionality they provide, not the sort of thing they contain" (it's the same as "no util package" imo)

Reply:
beyond the scope of this PR.